src/constants/geoChart.ts

```diff
@@ -3,6 +3,16 @@ import mapboxgl from "mapbox-gl";
 import { DEFAULT_COLORS } from "../components/visualizations/utils/color";
 import { IGeoLngLat } from "../interfaces/GeoChart";
 
+export const VIEWPORTS: { [key: string]: mapboxgl.LngLatBoundsLike } = {
```

Comment:
should seperate `{ [key: string]: mapboxgl.LngLatBoundsLike } `to type, such as `IGeoViewportConfig`

Reply:
We already have `IGeoConfigViewport`, so I use `IGeoViewports` instead :)

Reply:
updated. thanks :)